### PR TITLE
AMA-674 - reduce fragmentation by merging lessons

### PR DESCRIPTION
**Pull request by NgoniChoga**

### CODE REVIEW

**Review by TurRil — APPROVED** [review]

File: config/conversion_issues.json [hunk 1]
```diff
@@ -180,6 +180,23 @@
         "a": "more info"
       }
     },
+    {
+      "key": "lessons_attached_content",
```
Comment (smarquard) [inline config/conversion_issues.json:184]: embedded, not attached

File: config/conversion_issues.json [hunk 2]
```diff
@@ -180,6 +180,23 @@
...
+      "active": true,
+      "tool": "sakai.lessonbuildertool",
+      "icon-class": "icon-edit",
+      "description": "Lessons attached content",
+      "detail-heading": "Lessons pages",
+      "found": "Some Lessons pages have attachments that could not be moved automatically. These have been identified and placeholder put in place. Review and update this content.",
```
Comment (smarquard) [inline config/conversion_issues.json:190]: embedded not attachments, but needs a clearer description. @TurRil perhaps Thomas can suggest better wording.

File: run_workflow.py [hunk 3]
```diff
@@ -331,6 +331,9 @@ def start_workflow(link_id, site_id, APP):
                     if 'state' in step:
                         state = step['state']
 
+                    if 'flagged' in step and not step['flagged']:
```
Comment (smarquard) [inline run_workflow.py:334]: What is this doing exactly?

File: config/conversion_issues.json [hunk 4]
```diff
@@ -180,6 +180,23 @@
...
+      "not_found": "Good to go! No Vula lessons attachments found.",
+      "args": [
+        "lessons_soup"
+      ],
+      "info": {
+        "url": "https://docs.google.com/document/d/1m3xSY5gHPZ-Pw2KQcf5vcg73kW7CyuqVFI4fCl7mvvk/edit#heading=h.kszd2wgf91v7",
```
Comment (smarquard) [inline config/conversion_issues.json:196]: Doesn't seem to go to the right section if there is one.

**Comment by smarquard** [comment]
Added some comments but as this is merged already, please address them in a followup PR.

File: run_workflow.py (hunk 3 above)
Comment (NgoniChoga) [reply run_workflow.py:334]: Hey @smarquard, that is correct, it is like a feature flag. Just a flag to help us decide whether to run a workflow step or not to.

**Pull request merged**